pkgs/top-level/cuda-packages.nix

```diff
@@ -81,6 +83,45 @@ let
     nccl = final.callPackage ../development/cuda-modules/nccl { };
     nccl-tests = final.callPackage ../development/cuda-modules/nccl-tests { };
 
+    tests =
+      let
+        bools = [
+          true
+          false
+        ];
+        configs = attrsets.genAttrs [
+          "openCVFirst"
```

Comment:
replacing this with `order = [ ["opencv" "torch"] ["torch" "opencv"] ]` might allow further unnesting in `builder` below, but honestly I expect this to be modified and moved around more than once in the future so it really is ok to merge?

Reply:
Skipping for now; it'll likely be changed in the future so I think it's fine.